src/components/movies/movies.tsx

```diff
@@ -89,16 +93,23 @@ const form: AddEditFormData = {
 
 interface Props {
   className: string;
+  onMovieCardClick?: (e: any) => void;
 }
 
-const filter = ["all", "documentary", "comedy", "horror", "crime"];
+const filter = ["all", "documentary", "comedy", "horror", "cartoon"];
 const sort = ["Date", "Genre", "Title"];
 const menuOptions = ["Edit", "Delete"];
 
-const MoviesContainerComponent = ({ className }: Props) => {
+const MoviesContainerComponent = ({ className, onMovieCardClick }: Props) => {
   const [isModalVisible, setIsModalVisible] = useState(false);
   const [formData, setFormData] = useState(form);
   const [isDeleteMovie, setDeleteMovie] = useState(false);
+  const [filteredValue, setFilteredValue] = useState(null);
+  const [moviesList, setMoviesList] = useState([]);
+
+  useEffect(() => {
+    setMoviesList(movies);
+  }, [setMoviesList]);
```

Comment:
You don't need to pass `setState` function from `useState` because React guarantees it's identity on re-renders.
[https://ru.reactjs.org/docs/hooks-reference.html#usestate](url)